src/builder.js

```diff
@@ -131,6 +131,7 @@ builder.build = function(project, uuid, path, gitBranch, branch, dockerOptions)
   var buildId     = imageId + ':' + branch;
   var author      = 'unknown@unknown.com';
   var now         = moment();
+  var sha         = '';
```

Comment:
currently `sha` is global and on concurrent builds GitHub status doesn't work, Roger share `sha` values across different builds